.github/workflows/check.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           # Specify `--manifest-path` as a workaround.
           #
           # See https://github.com/actions-rs/clippy-check/issues/28
-          args: --all-features --workspace --manifest-path rust/Cargo.toml
+          args: --all-features --workspace --manifest-path rust/Cargo.toml -- -D warnings
```

Comment:
This makes the clippy check [more strict](https://doc.rust-lang.org/stable/clippy/usage.html#command-line) — it will fail if any warnings are found.